Core/src/org/sleuthkit/autopsy/contentviewers/AnnotationsContentViewer.java

```diff
@@ -0,0 +1,446 @@
+     */
+    private void endTable(StringBuilder html) {
+        html.append("</table><br><br>"); //NON-NLS
+    }
+
+    /**
+     * End a data section.
+     * 
+     * @param html The HTML text on which to end a section.
+     */
+    private void endSection(StringBuilder html) {
+        html.append("<br>"); //NON-NLS
+    }
+    
+    /**
+     * Apply escape sequence to special characters. Line feed and carriage
+     * return character combinations will be converted to HTML line breaks.
+     * 
+     * @param text The text to format.
+     * @return The formatted text.
+     */
+    private String formatHtmlString(String text) {
+        String formattedString = StringEscapeUtils.escapeHtml4(text);
+        return formattedString.replaceAll("(\r\n|\r|\n|\n\r)", "<br>");
+    }
+
+    /**
+     * This method is called from within the constructor to initialize the form.
+     * WARNING: Do NOT modify this code. The content of this method is always
+     * regenerated by the Form Editor.
+     */
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    private void initComponents() {
+
+        jScrollPane5 = new javax.swing.JScrollPane();
+        jTextPane1 = new javax.swing.JTextPane();
+
+        setPreferredSize(new java.awt.Dimension(100, 58));
+
+        jTextPane1.setEditable(false);
+        jTextPane1.setName(""); // NOI18N
+        jTextPane1.setPreferredSize(new java.awt.Dimension(600, 52));
+        jScrollPane5.setViewportView(jTextPane1);
+
+        javax.swing.GroupLayout layout = new javax.swing.GroupLayout(this);
+        this.setLayout(layout);
+        layout.setHorizontalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addComponent(jScrollPane5, javax.swing.GroupLayout.DEFAULT_SIZE, 907, Short.MAX_VALUE)
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addComponent(jScrollPane5, javax.swing.GroupLayout.Alignment.TRAILING, javax.swing.GroupLayout.DEFAULT_SIZE, 435, Short.MAX_VALUE)
+        );
+    }// </editor-fold>//GEN-END:initComponents
+
+    // Variables declaration - do not modify//GEN-BEGIN:variables
+    private javax.swing.JScrollPane jScrollPane5;
+    private javax.swing.JTextPane jTextPane1;
+    // End of variables declaration//GEN-END:variables
+
+    @Override
+    public String getTitle() {
+        return Bundle.AnnotationsContentViewer_title();
+    }
+
+    @Override
+    public String getToolTip() {
+        return Bundle.AnnotationsContentViewer_toolTip();
+    }
+
+    @Override
+    public DataContentViewer createInstance() {
+        return new AnnotationsContentViewer();
+    }
+
+    @Override
+    public boolean isSupported(Node node) {
+        AbstractFile file = node.getLookup().lookup(AbstractFile.class);
+        return file != null;
+    }
```

Comment:
Do we want this disabled for Local Directories and Virtual Directories neither of which we have any way to comment on currently to my knowledge.